spec/model/car_cleaner_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../../model/car_cleaner'
+
+describe CarCleaner do
+  
+  context 'with empty data' do
+    subject { CarCleaner.new }
+
+    it 'stores an array as a queue' do
+      response = subject
+
+      expect(response.queue).to eq []
+    end
+
+  end
+
+  context 'with data' do
+    subject { CarCleaner.new(['Test', 'Another Test']) }
+  
+    it 'stores an array as a queue' do
+      response = subject
+      
+      expect(response.queue). to eq ['Test', 'Another Test']
+    end
+
+  end
+  
+end
```

Comment:
I noticed you use to add a newline before the last `end`. I don't think that's necessary. it's not wrong or anything, but it's not necessary and rubocop will complain about that

Reply:
I will remove the redundant lines.